src/test-run/index.js

```diff
@@ -247,5 +255,9 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
     if (this.lastDriverStatusResponse)
         return this.lastDriverStatusResponse;
 
-    return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
+    // NOTE: browsers abort opened xhr request after some timeout (timeout depends on the browser).
+    // To avoid this we send an empty response after 2 minutes if we didn't get any command.
```

Comment:
```
    // NOTE: browsers abort an opened xhr request after a certain timeout (the actual duration depends on the browser).
    // To avoid this, we send an empty response after 2 minutes if we didn't get any command.
```